aws/table_aws_glue_job.go

```diff
@@ -0,0 +1,287 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/glue"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsGlueJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_glue_job",
+		Description: "AWS Glue Job",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"EntityNotFoundException"}),
+			},
+			Hydrate: getGlueJob,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listGlueJobs,
```

Comment:
Do we have any optional quals support for it?

Reply:
Only pagination params are there